app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
+    render template: "posts/index.html.erb", layout: "application"
```

Comment:
This line isn't needed, when you setup routes, ruby automatically renders the template.